docs/contributors/patching/index.md

```diff
@@ -10,4 +10,5 @@ Work with Debian patches <work-with-debian-patches>
 Commit changes <commit-changes>
 Submit a merge proposal <submit-a-merge-proposal>
 Dual maintenance with Salsa and git-ubuntu <dual-maintenance-with-salsa-and-git-ubuntu>
+maintain-ubuntu-desktop-software
```

Comment:
Given that at this point, this workflow differs from the 'default' package maintenance guidance as described elsewhere in this docs set, I'd say this article (and the associated set-up article) should be under [Niche package maintenance](https://documentation.ubuntu.com/project/maintainers/niche-package-maintenance/) as "Ubuntu Desktop packaging".

If/when Git Build Package gets adopted as the default workflow, this entire docs set would need to be rewritten, and these Desktop-specific instructions would be incorporated into it.